crates/network-scheduler/src/assignment.rs

```diff
@@ -154,15 +177,34 @@ impl Assignment {
     }
 
     pub fn regenerate_headers(&mut self, cloudflare_storage_secret: String) {
+        let alice_secret_key = SecretKey::generate(&mut OsRng);
```

Comment:
Let's just call it temp key and worker key before committing

Reply:
will do